pkg/auth/auth_test.go

```diff
@@ -6,37 +6,9 @@
 package auth
 
 import (
-	"reflect"
-	"strings"
 	"testing"
-
-	"github.com/Azure/kubernetes-kms/pkg/config"
-
-	"github.com/Azure/go-autorest/autorest/adal"
-	"github.com/Azure/go-autorest/autorest/azure"
 )
 
-func TestParseAzureEnvironment(t *testing.T) {
```

Comment:
A lot of the existing auth mechanism from upstream seem to have been broken by this PR and this will definitely impact the use of this fork by OCP